src/sdks/tableau/apis/viewsApi.ts

```diff
@@ -85,7 +85,7 @@ const queryViewsForSiteEndpoint = makeEndpoint({
   ],
   response: z.object({
     pagination: paginationSchema,
-    views: z.object({ view: z.array(viewSchema) }),
+    views: z.object({ view: z.array(viewSchema).optional() }),
```

Comment:
I noticed that if you, say, provide a bad filter, Tableau doesn't return an empty array... `view` is undefined.